src/shadowbox/server/manager_service.ts

```diff
@@ -52,12 +52,12 @@ interface RequestParams {
   id?: string;
   name?: string;
   metricsEnabled?: boolean;
-  limit?: DataUsage|{};
+  limit?: DataUsage;
   port?: number;
   hours?: number;
 }
 interface RequestType {
-  params: RequestParams;
+  params: RequestParams|{};
```

Comment:
If we make that change to `RequestParams`, then `params` doesn't need the `|{}`.